src/neo/SmartContract/Native/NameService.cs

```diff
@@ -53,6 +53,11 @@ internal override void OnPersist(ApplicationEngine engine)
             }
         }
 
+        protected override void OnTransferred(ApplicationEngine engine, UInt160 from, NameState token)
+        {
+            token.Admin = null;
```

Comment:
Set Admin to `to`?

Reply:
`to` becomes to `Owner`, and we set `Admin` to `null`.